src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+function addSpan([...elements]) {
+  elements.forEach(item => {
+    const listItem = item.querySelector('ul');
+
+    if (item.children.length > 0) {
+      const textHead = item.firstChild.textContent.trim();
+      const span = document.createElement('span');
+
+      span.innerText = `${textHead}`;
+      item.firstChild.textContent = '';
+      item.prepend(span);
+
+      span.addEventListener('click', (e) => {
+        listItem.hidden = listItem.hidden = !listItem.hidden;
```

Comment:
There is a typo in the assignment, you should only assign the new value once: `listItem.hidden = !listItem.hidden;`